src/service/notification-service.ts

```diff
@@ -0,0 +1,33 @@
+import { LocalNotifications } from '@capacitor/local-notifications';
+
+export class NotificationService {
+    /**
+     * Request permission from the user to show notifications
+     * It's necessary to use the sendNotification method
+     */
+    authorize = async () => {
+        await LocalNotifications.requestPermissions()
+    }
+
+    /**
+     * Send a local notification to the user after 5 seconds 
+     * @param title - the title of the notification 
+     * @param body - the body of the notification
+     */
+    sendNotification = async (title: string, body: string) => {
+        await LocalNotifications.schedule({
```

Comment:
The `id` field should be a number that we can default to a rand. If we use the same id for every notification, the user won't get different types of notifications. 

The `schedule` fields should also be a function parameter, which we can have a default (but I'd think the default should be immediately, not 5s from now). 

I would test/check on the setting the actionTypeId to an empty string, this doesn't seem correct after quickly looking at the docs.

Reply:
updated the id and schedule!

the `actionTypeId` its used as an empty string in the documentation and it does not gives a description for it 

https://capacitorjs.com/docs/v2/apis/local-notifications#example